blocks/init/src/Blocks/components/image/manifest.json

```diff
@@ -37,6 +38,9 @@
 			"type": "boolean",
 			"default": false
 		},
+		"imageAlt": {
```

Comment:
If added, this should probably be set in the block as well (taken from the image's alt tag and overridable) but probably out of scope for this PR.

Reply:
Not sure will check it out. Block should be the place where alt tag is set (passed to the component)